src/docker/.env

```diff
@@ -1,7 +1,21 @@
 #BASE_PATH=dashboard/
 #BASE_URL=http://localhost:8501/dashboard
 
-#GRAPHISTRY_USERNAME=abc
-#GRAPHISTRY_PASSWORD=xyz
 
-#LOG_LEVEL=DEBUG
+GRAPHISTRY_USERNAME=tcookaustin
+GRAPHISTRY_PASSWORD=p2ys3Eg0ykM&Oep8
+GRAPHISTRY_PROTOCOL=https
+GRAPHISTRY_CLIENT_HOSTNAME=""
+GRAPHISTRY_SERVER="hub.graphistry.com"
+
+# Splunk credentials
+SPLUNK_USERNAME="tcook-reader"
+SPLUNK_PASSWORD="nC&88%0fl^CEW"
+SPLUNK_HOST="splunk.graphistry.com"
+
+
+# App configuration
+PIVOT_URL_INVESTIGATION_ID="123"
+APP_NOW_TIME="2019-03-18T00:00:00Z"
```

Comment:
remove these too? can be in the view

Reply:
Not sure what you mean by "in the view" ?  Don't the envars have to be passed thru .env?

Reply:
why make these global env vars, vs just some constant in `app/views/demo_avr/__init__.py` ?